pkgs/applications/video/mpv/default.nix

```diff
@@ -45,6 +45,7 @@
 , swiftSupport       ? false,          swift         ? null
 , theoraSupport      ? true,           libtheora     ? null
 , vaapiSupport       ? stdenv.isLinux, libva         ? null
+, sixelSupport       ? true,           libsixel      ? null
```

Comment:
To keep the list sorted, we could put this a few lines above, just below `sdl2Support`

Reply:
I would like to disable sixelSupport by default cause not every terminal supports and some like Kitty even do a better job themselves.

Reply:
@SuperSandro2000 I'm actually wondering if we want to provide a second package (e.g. `mpvFull`) that has all features enabled for people who don't care about size etc., but about having all features. Something similar happens e.g. with `diffoscope`, `curl` or ` python`. Just grep for e.g. `Full` on `all-packages.nix` to get a few more examples.

@prusnak while I appreciate everyone who invests free time in helping out maintaining `nixpkgs`, I'd like to mention here that I'm not a big fan of those low-impact nitpicks that don't really have an actual benefit and can be taken up by either a tool or the next time someone touches the package. You may have noticed that we had several policy discussions in the past regarding the contribution process and one of the issues a lot of people have is that our PR queue is steadily growing and a lot of PRs stall because of that. Feel free to disagree (I mean, I want to mention it here, but I don't intend to speak speak for the project and there's no actual policy around that), but I think that those sort of reviews kind of help slowing down the merge process.

Reply:
@prusnak Reordered the fields

Reply:
> I would like to disable sixelSupport by default cause not every terminal supports and some like Kitty even do a better job themselves.

I tested Kitty 0.19.0 (with no user configuration) and mpv's sixel output didn't work with it ¯\\\_(ツ)\_/¯